.github/workflows/doctests.yml

```diff
@@ -0,0 +1,33 @@
+name: doctests
+on:
+  push:
+    branches:
+      - 'main'
+      - /^release-.*$/
+    tags: '*'
+  pull_request:
+jobs:
+  doctests:
+    name: Doctests (Julia ${{ matrix.julia-version }} - ${{ github.event_name }})
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        julia-version: [1.6.1]
+    steps:
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: ${{ matrix.julia-version }}
+      - uses: actions/checkout@v1
+      - name: Install dependencies
+        run: julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()'
+      - name: Fix doctests
+        run: julia --code-coverage --project=docs 'docs/fix_doctests.jl'
```

Comment:
What does `--code-coverage` actually do? 

```
 --code-coverage={none|user|all}, --code-coverage
                           Count executions of source lines (omitting setting is equivalent to "user")
```

It's not super clear...

Reply:
I don’t know aha. I copied this from Pkg, it seems this is the choice of flag it makes when running tests